frontend/packages/knative-plugin/src/components/overview/__tests__/OverviewDetailsKnativeResourcesTab.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { OverviewItem } from '@console/shared';
+import { MockKnativeResources } from '@console/dev-console/src/components/topology/__tests__/topology-knative-test-data';
+import OverviewDetailsKnativeResourcesTab from '../OverviewDetailsKnativeResourcesTab';
+import KnativeServiceResources from '../KnativeServiceResources';
+import KnativeRevisionResources from '../KnativeRevisionResources';
+import EventSinkServicesOverviewList from '../EventSinkServicesOverviewList';
+
+describe('OverviewDetailsKnativeResourcesTab', () => {
+  it('should render KnativeServiceResources on sidebar', () => {
+    const svcItem: OverviewItem = {
```

Comment:
Define the props type using ```React.ComponentProps```. Avoid importing the prop type in spec file.

Reply:
Thanks :)